### DISCUSSION TITLE
Duda CSS

### DISCUSSION THREAD

**Discussion by OscarIslasH** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#378] [Q&A] [answered]
¿Como podria posicionar el logo de vaccination a la izquierda y los demas botones a la derecha? Me sale asi:

![Captura de practica 3](https://user-images.githubusercontent.com/99118114/158042804-d273ccf9-74fc-4939-9943-68b4a6182d92.png)

Ya probe poniendo el " text-align: left; " " padding-left " y el " z-indez " y no se si los estoy poniendo donde no van o si en general los esta afectando otra cosa

Aca dejo el codigo del HTML:

![Captura de pantalla 2022-03-12 204814](https://user-images.githubusercontent.com/99118114/158042981-9e99a23c-d3dc-4bef-b862-e1b5cb35c0dc.png)

Aca dejo el codigo de CSS:

![Captura de pantalla 2022-03-12 204833](https://user-images.githubusercontent.com/99118114/158043001-5b952e08-6fcb-49df-bf92-3e94b890000f.png)

![Captura de pantalla 2022-03-12 204852](https://user-images.githubusercontent.com/99118114/158043003-559eb1b6-8156-4540-9139-e8342a3e662e.png)

Cualquier observacion es bienvenida, apenas empece hoy a ver lo de css, por lo que no se mucho.

**Comment by AngelCruzO** [comment]
a tu clase header le puedes poner un display: flex; justify-content: space-between o space-around; align-item-center; normalmente asi acomodo mis menus

**Reply by OscarIslasH** [reply]
¿Pero como acomodo lo azul a la izquierda y lo de negro hacia la derecha?

**Reply by OmarVenturaP** [reply]
Asi como te explico el compañero, con esos atributos logras hacer el efecto de espaciado que buscas.

**Reply by AngelCruzO** [reply]
![image](https://user-images.githubusercontent.com/88981492/158046783-971d5223-fbfc-457f-bbf2-5a1e5dc93486.png)

En mi caso yo ocupo directamente la etiqueta nav con estos estilos, asi se hace la separación entre el logo y los enlaces

**Reply by OscarIslasH** [reply]
Si lo habia puesto, solo que lo puse mal, le puse las propiedades a una clase abajo que controlaba tambien el logo, queriendo mover solo lo de la barra de navegacion afectaba tambien el logo: 


.contenedor {
    width: 100%;
    max-width: 1080px;
    margin: 0 auto;
}


.header {
    background-color: rgb(255, 255, 255);
    color: white;
    padding: 30px;
    background-size: 10px; 
    font-size: 30px;
    padding: 65px ;
    position: fixed;
    text-decoration: none;
    display: flex; 
    justify-content: space-between; 
    align-items: center;
    
}

.header .logo {
    color: #3146ff;
    padding-left: 100px; 
    letter-spacing: 1px; 
    font-size: 35px;
    color: #3146ff;
    
}

.header a {
    color: rgb(66, 65, 65);
    text-decoration: none;
    box-sizing: border-box;
   
    
}

.logo-barra-de-navegacion{
    display: flex;
    justify-content: space-between;
    align-items: center;
    border: 0em;
    height: ;

}

**Reply by OscarIslasH** [reply]
Y el codigo deberia de estar escrito asi: 

.header {
    background-color: rgb(255, 255, 255);
    color: white;
    padding: 30px;
    background-size: 10px; 
    font-size: 30px;
    padding: 65px ;
    position: fixed;
    text-decoration: none;
    display: flex; 
    justify-content: space-between; 
    align-items: center;
    
}

.contenedor {
    width: 100%;
    max-width: 1080px;
    margin: 0 auto;
}

.logo-barra-de-navegacion{
    display: flex;
    justify-content: space-between;
    align-items: center;
    border: 0em;

}

.logo {
    color: #3146ff;
    padding-left: 100px; 
    letter-spacing: 1px; 
    font-size: 35px;
    color: #3146ff;
    
}

.Barra-de-navegacion {
    color: rgb(66, 65, 65);
    text-decoration: none;
    box-sizing: border-box;
    position : relative;
    Left :40% ;
    margin: 0%;
    padding: 0%;
    list-style: none;
}

**Comment by Ingrzs** [comment, marked as answer]
A la clase .logo dale position absolute o relative
.logo {
position : relative;
Left :10% ; les vas calculando (si gira al contrario entonces ponle Right)

Y lo mismo haces con la clase de barra navegacion.

Ojo  a las dos clases les debes dar la misma propiedad de position, de lo contrario en una te va a funcionar en la otra no.

**Reply by OscarIslasH** [reply]
Muchas gracias, recien me di cuenta cuando estaba acomodando las cosas que hacia muchas clases que sobrescribian otras, al poner lo que me dijiste, se movian tanto el logo y la barra de navegacion, asi me di cuenta. 

Muchas gracias, en serio me estaba estresando bastante por que no entendia por que no se movia.

**Reply by Ingrzs** [reply]
@OscarIslasH 
Si tienes tu lista con un display flex
Prueba poniéndole 
Justify-content: space between o space-around. (Recuerda el espacio se adaptara al tamaño del contenedor dinde tengas tu lista)

Si no prueba poniéndole margin-lef: 5% 
Ahi calcularas cuando espacios quieres entre las listas.

Y yo siempre le pongo un border a cada elemento para saber hasta dónde abarca su tamaño y si se mueve o no, ahi se quien afecta a quien, Saludos y éxitos.
